addons/website/i18n/website.pot

```diff
@@ -7623,6 +7645,13 @@ msgstr ""
 msgid "Primary"
 msgstr ""
 
+#. module: website
+#. odoo-javascript
+#: code:addons/website/static/src/xml/website.editor.xml:0
+#, python-format
+msgid "Primary Buttons"
+msgstr ""
+
```

Comment:
I would limit the diff of this PR to only the wording change you fix, instead of modifying / syncing the whole .pot

Reply:
Makes sense. I removed the other diff.